sites/public/src/pages/applications/contact/address.tsx

```diff
@@ -514,7 +514,8 @@ const ApplicationAddress = () => {
                     errors?.applicant?.workInRegion ? "text-alert" : ""
                   }`}
                 >
-                  {t("application.contact.doYouWorkIn", { county: listing?.countyCode })}
+                  {/* TODO: where can I get the county code from? */}
+                  {/* {t("application.contact.doYouWorkIn", { county: listing?.countyCode })} */}
```

Comment:
do you still need these comments? I thought we could use jurisidiction.name here?

Reply:
Good catch, this has been updated